client/lib/cart/store/cart-analytics.js

```diff
@@ -17,6 +17,8 @@ function recordEvents( previousCart, nextCart ) {
 
 function recordAddEvent( cartItem ) {
 	analytics.tracks.recordEvent( 'calypso_cart_product_add', cartItem );
+	analytics.ga.recordEvent( 'Checkout', 'calypso_cart_product_add',
+		`${ cartItem.product_id } ${ cartItem.product_slug }` );
```

Comment:
@rralian is `product_id` or `product_slug` more useful? It'll probably be easier to slice on this dimension if we only use one or the other.